isisd/isis_pdu.c

```diff
@@ -205,45 +250,35 @@ static int process_p2p_hello(struct iih_info *iih)
 	/* 8.2.5.2 a) a match was detected */
 	if (isis_tlvs_area_addresses_match(iih->tlvs,
 					   iih->circuit->area->area_addrs)) {
-		/* 8.2.5.2 a) 2) If the system is L1 - table 5 */
-		if (iih->circuit->area->is_type == IS_LEVEL_1) {
+		/* 8.2.5.2 a) 2) If the calculated type is L1 - table 5 */
+		if (iih->calculated_type == IS_LEVEL_1) {
+		//if (iih->circuit->area->is_type == IS_LEVEL_1) {
```

Comment:
Please do not leave dead code in the commit additionally c++ style comments should not be used.